### DISCUSSION TITLE
How to replicate "layout glued" to keyboard on MAUI.iOS

### DISCUSSION THREAD

**Discussion by Pastajello** [dotnet/maui#22080] [Upgrading from Xamarin]
Hello,

I'm migrating an application from XF to MAUI. The application have around 20k users and Im about 2 serious issues away from completing the migration.

The issue I'm asking for help resolving is this:

https://github.com/dotnet/maui/assets/28345098/779a51ba-5f7a-4cc5-9617-4876a8d5fbfc

How do I make a layout that gues itself to the keyboard, when the keyboard goes up? I've tried with Grrid, StackLayout, VerticalStackLayout, nothing is working.
I want to point that I know, that on MAUI we should not put scrollView in StackLayout, I just want to demonstrate the difference in behavior.

I've created a sample project with the above demonstration. The code in both is similar, the difference is that XF uses renderer on iOS, on MAUI I'm using the iOS code on the page itself.

https://github.com/Pastajello/MAUIKeyboardProblems

Help :(

**Comment by davidortinau** [comment]
I think you want to do an input accessory view.

Sample: 
[KeyboardAccessoryView.zip](https://github.com/user-attachments/files/15567724/KeyboardAccessoryView.zip)

https://github.com/dotnet/maui/assets/41873/93a4045a-e753-46fa-b5d0-c357f4d6a497

https://dev.to/vhugogarcia/done-button-to-keyboard-in-net-maui-3211
